diff --git a/node_modules/react-native-ultimate-config/android/build.gradle b/node_modules/react-native-ultimate-config/android/build.gradle
index 7ba78ad..24b72de 100644
--- a/node_modules/react-native-ultimate-config/android/build.gradle
+++ b/node_modules/react-native-ultimate-config/android/build.gradle
@@ -109,10 +109,9 @@ afterEvaluate { project ->
     task androidJavadoc(type: Javadoc) {
         source = android.sourceSets.main.java.srcDirs
         classpath += files(android.bootClasspath)
-        project.getConfigurations().implementation.setCanBeResolved(true)
-        classpath += files(project.getConfigurations().getByName('implementation').asList())
+//         project.getConfigurations().implementation.setCanBeResolved(true)
+//         classpath += files(project.getConfigurations().getByName('implementation').asList())
         include '**/*.java'
-
     }
 
     task androidJavadocJar(type: Jar, dependsOn: androidJavadoc) {
